libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -101,9 +101,13 @@ public static String getIpvSessionId(JourneyRequest request, boolean allowNull)
     }
 
     public static List<String> getFeatureSet(JourneyRequest request) {
-        List<String> featureSet = request.getFeatureSet();
-        LogHelper.attachFeatureSetToLogs(featureSet);
-        return (featureSet != null) ? featureSet : Collections.emptyList();
+        String featureSet = request.getFeatureSet();
+        List<String> featureSetList =
+                (featureSet != null && !featureSet.isBlank())
+                        ? Arrays.asList(featureSet.split(","))
+                        : Collections.emptyList();
```

Comment:
(Still feels as though there's potential to pull this out as a method - we've repeated it 4 or 5 times)